src/test/java/org/folio/rest/impl/AutomatedPatronBlocksAPITest.java

```diff
@@ -79,6 +78,10 @@
     LIMIT_VALUES.put(MAX_OUTSTANDING_FEE_FINE_BALANCE, 70);
   }
 
+  private boolean expectBlockBorrowing = false;
+  private boolean expectBlockRenewals = false;
+  private boolean expectBlockRequests = false;
```

Comment:
should we initialize these in `beforeEach()`?

Reply:
Done